.github/workflows/build.yml

```diff
@@ -2,20 +2,31 @@ name: Build
 
 on:
   workflow_dispatch:
+    inputs:
+      skip-duplicates:
+          description: 'Whether to fail or skip duplicates when uploading to a package repository'
+          required: false
+          default: 'true'
   push:
-    branches: [ beta, master, release-* ]
+    branches: [ beta, master, release-*, push ]
   pull_request:
     branches: [ master, release-* ]
 
 jobs:
   build:
-    env:
+    env:    
       GIT_REF: ${{ github.ref }}
       Solution: src/GeneXus.DeploymentTargets.sln
       Configuration: Release
 
     runs-on: [windows-latest]
 
+    outputs:
+      NUGET_VERSION: ${{ steps.buildVariables.outputs.NUGET_PACKAGE_VERSION }}
+      LAST_COMMITTER: ${{ steps.buildVariables.outputs.LAST_COMMITTER }}
+      COMMIT_MESSAGE: ${{ steps.buildVariables.outputs.COMMIT_MESSAGE }}
+      SHOULD_DEPLOY: ${{ steps.buildVariables.outputs.SHOULD_DEPLOY }}
+      
```

Comment:
A step with id "buildVariables" is missing where these 4 values should be calculated